src/factory_sim/description/bracket_collision_geometry.xml

```diff
@@ -0,0 +1,92 @@
+<mujoco model="bracket_collision_geometry">
+  <!-- 5 Box collision primitives -->
+  <!-- starting from the left (pointy end) -->
+  <!-- top cheek -->
+  <body>
```

Comment:
you need to push up the change to delete this body tag

Reply:
On my current branch I actually moved it in to this xml and out of the scene.xml to make it more like suction_tool.xml https://github.com/PickNikRobotics/moveit_pro_example_ws/blob/13300-test-icp/src/factory_sim/description/bracket_collision_geometry.xml#L6-L7

I've also added some urdfs so they can be loaded into the planning scene and a urdf of the bin table. Should I add all that here?

and should I reduce the parts to three?

Reply:
Feels like stuff like the urdf of bin and table should be part of the PR for https://github.com/PickNikRobotics/moveit_pro/issues/13372

I would just add 3 since that is how v1 of the objective works.